.github/workflows/cloudbeat-ci.yml

```diff
@@ -5,7 +5,7 @@ on:
     branches:
       - main
       - '[0-9]+.[0-9]+'
-    types: [opened, synchronize, reopened]
+    types: [ opened, synchronize, reopened ]
```

Comment:
is padding needed here or is it just for formatting?

Reply:
automated formatting done by the IDE, i'll remove the WF change from the PR as it was just a test to see if this solves the issue